### DISCUSSION TITLE
Add "Intent" options for Android and IOS （UNI）

### DISCUSSION THREAD

**Discussion by kb9gxk** [rustdesk/rustdesk#4669] [Q&A] [answered]
Please add the intent options to the Android and IOS manifests.  On windows, I can have an HTML file that calls "rustdesk://connections/new/<ID>" and it will prompt to open RustDesk with that ID

**Comment by kb9gxk** [comment]
I see the new format is "rustdesk://connection/[ID]" but this is still not set as an intent for Android or IOS.

I created an HTML file to create links to my clients and it works great for Windows, but need the ability on my Android and Chromebook devices to just click my link to connect.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/1330#discussioncomment-6860095, also start server via intent

**Comment by rustdesk** [comment]
add to milestone. https://github.com/rustdesk/rustdesk/discussions/918

**Comment by rustdesk** [comment]
@sahilyeole

**Comment by kb9gxk** [comment, marked as answer]
@sahilyeole added as Pull Request https://github.com/rustdesk/rustdesk/pull/6266 for inclusion in 1.2.4

**Comment by clanto007** [comment]
This merge is wonderful,

Is there the possibility to add also password in fields to auto connect directly?

As for desktop:
rustdesk://connection/new/{ID}?password={password}
